lib/resources/datav2/entityv2.ts

```diff
@@ -357,6 +358,7 @@ export interface CryptoBar {
 export interface RawCryptoBar {
   T: string;
   S: string;
+  x: string;
```

Comment:
Is this change in here to also [close out this PR](https://github.com/alpacahq/alpaca-trade-api-js/pull/190/files#r805775076)?

Reply:
Yes, I will close that after this one will be merged and published.